hack/retag-images.sh

```diff
@@ -19,27 +19,32 @@ set -euo pipefail
 TARGET_REGISTRY=${TARGET_REGISTRY:-127.0.0.1:5000}
 
 function retag {
-  local IMAGE="$1"
+  local image="$1"
 
-  ORG="$(echo ${IMAGE} | cut -d / -f2)"
-  NAME="$(echo ${IMAGE} | cut -d / -f3 | cut -d : -f1)"
-  TAG="$(echo ${IMAGE} | cut -d / -f3 | cut -d : -f2)"
-  TARGET_IMAGE="${TARGET_REGISTRY}/${ORG}/${NAME}:${TAG}"
+  # trim registry
+  local local_image="$(echo ${image} | cut -d/ -f1 --complement)"
```

Comment:
There is no such flag `--complement` on macos, if that's important

Reply:
Is MacOS for this script relevant? Also I kind of like the old variant (using cut instead of IFS) more readable than array magic. ^^

Reply:
I guess it's not important if  this script is intended to only run in CI

Reply:
Sooo... LGTY? ^^